mozci/push.py

```diff
@@ -375,6 +375,82 @@ def scheduled_duration(self):
 
         return int(duration / 3600)
 
+    def _check_classification(self, failure_push, classifications):
```

Comment:
Is it worth memoizing this?

Reply:
Probably not, as we will never call this twice with the same parameters given that we already memoize get_candidate_regressions